src/intelligence/providers/ai-wrapper.service.ts

```
@@ -155,6 +155,194 @@ export class AiWrapperService {
   }
 
+  /**
+   * Generate content with function calling capabilities
+   * This is a temporary implementation that simulates function calling
+   * by extracting function calls from the AI response text
+   */
+  async generateFunctionCallingContent(
+    model: AIModels,
+    prompt: string,
+    history: ChatHistory[],
+    functions: Record<string, Function>,
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)